Make fails on centos 6

Centos 6 using latest go in repository:

```
# make test
mkdir -p /lib/security/pam-ussh/.go/src
GOPATH=/lib/security/pam-ussh/.go go get golang.org/x/crypto/ssh
GOPATH=/lib/security/pam-ussh/.go go get golang.org/x/crypto/ssh/agent
GOPATH=/lib/security/pam-ussh/.go go get github.com/stretchr/testify/require
GOPATH=/lib/security/pam-ussh/.go go test -cover
# _/lib/security/pam-ussh
/tmp/go-build020625526/_/lib/security/pam-ussh/_test/_obj_test/pam_ussh.go:128: unknown ssh.CertChecker field 'IsAuthority' in struct literal
FAIL	_/lib/security/pam-ussh [build failed]
make: *** [test] Error 2
```

Same error on Centos 7.  
```
# make
mkdir -p /lib/security/pam-ussh/.go/src
GOPATH=/lib/security/pam-ussh/.go go get golang.org/x/crypto/ssh
GOPATH=/lib/security/pam-ussh/.go go get golang.org/x/crypto/ssh/agent
GOPATH=/lib/security/pam-ussh/.go go get github.com/stretchr/testify/require
GOPATH=/lib/security/pam-ussh/.go go test -cover
# _/lib/security/pam-ussh
/tmp/go-build444999842/_/lib/security/pam-ussh/_test/_obj_test/pam_ussh.go:121:14: unknown field 'IsAuthority' in struct literal of type ssh.CertChecker
FAIL	_/lib/security/pam-ussh [build failed]
make: *** [test] Error 2
```
Centos 7 also has the dependency:
```
yum install pam-devel
```

--- a/pam_ussh.go
+++ b/pam_ussh.go
@@ -124,7 +124,7 @@ func authenticate(w io.Writer, uid int, username, ca string, principals map[stri
 	}
 
 	c := &ssh.CertChecker{
-		IsAuthority: func(auth ssh.PublicKey) bool {
+		IsUserAuthority: func(auth ssh.PublicKey) bool {
 			return bytes.Equal(auth.Marshal(), caPubkey.Marshal())
 		},
 	}

